app/api/zora/score/route.tsx

```diff
@@ -2,25 +2,43 @@ import { NextRequest } from 'next/server';
 import { Address } from 'viem';
 import trackEndpoint from '@/lib/stack/trackEndpoint';
 import getZoraProfileScore from '@/lib/zora/score/getZoraProfileScore';
-import { EVENT_ZORA_SCORE } from '@/lib/consts';
+import { EVENT_ZORA_SCORE, MAX_CREATE_SCORE, TOKEN_INDEXER_POINT_ID } from '@/lib/consts';
 import getZoraScore from '@/lib/zora/score/getZoraScore';
+import { createStackClient } from '@/lib/stack/client';
+
+const maxCreateScore = MAX_CREATE_SCORE;
+const stack = createStackClient(TOKEN_INDEXER_POINT_ID);
+
+function calculateCreateScore(tokensCreated: number) {
+  const score = Math.min(tokensCreated / maxCreateScore, 1);
+  return score * 100;
+}
```

Comment:
each file should do `precisely` one thing.
- actual: this file is defining `calculateCreateScore` AND `API` route.
- expected: `route.tsx` only contains the API route definition. Other functions are imported from the `lib/` directory.

action: please create a new lib called `lib/calculateCreateScore.tsx`